src/Tracy/Debugger.php

```diff
@@ -135,6 +141,13 @@ public static function enable($mode = NULL, $logDirectory = NULL, $email = NULL)
 		self::$obLevel = ob_get_level();
 		error_reporting(E_ALL);
 
+		// get initial rusage for CPU output in info panel
+		self::$getrusage = function_exists('getrusage');
+		if(self::$getrusage) {
+			$dat = getrusage();
+			self::$rusage = $dat["ru_utime.tv_sec"]*1e6+$dat["ru_utime.tv_usec"];
+		}
```

Comment:
Shouldn't this be skipped in production mode? Also your coding style is not correct.